src/Repository/LogActionItemRepository.php

```diff
@@ -90,4 +90,51 @@ public function countDownloads(\DateTime $start = null, \DateTime $stop = null):
 
         return $downloadCount;
     }
+
+    /**
+     * Array downloads, optional date range and also filter aware.
+     *
+     * @return array the count of the datasets downloaded, per FAIR guidelines
+     */
+    public function getDownloads(): array
```

Comment:
How is this function different from CountDownloads?

Reply:
Moved to the report.